.github/workflows/main.yml

```diff
@@ -92,3 +92,25 @@ jobs:
       # The Windows CI transforms the `c-library` symlink into a real directory, modifying the Git
       # state, so we ignore these changes with `--allow-dirty` here.
       run: scripts/verify-publish.sh --allow-dirty
+
+  itt_python_build_and_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+          - os: windows-latest
+    runs-on: ['${{matrix.os}}']
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+      - name: Build and Install ittapi
+        run: |
+          cd python
```

Comment:
instead using "cd python" here and below it would be better to specify the default working directory for the job.
```
    defaults:
      run:
        working-directory: rust
```